### ISSUE TITLE
Problem not reading folder responsivefilemanager

### ISSUE THREAD

**Issue by zoheirsoftware** [gordon-matt/peachpie-responsive-file-manager#22]
Hi
I created the project exactly like you
I run the following error when running

Failed to load: https://localhost:44377/js/tinymce/plugins/responsivefilemanager/plugin.js

You can download the link of my project that I created

https://s17.picofile.com/file/8418694026/WebApplication12.zip.html

Please check and advise how to fix the problem

Thanks

**Comment by gordon-matt** [comment]
Why do you copy the projects into your solution? Just use the NuGet package. Reference this: https://www.nuget.org/packages/ResponsiveFileManager.AspNetCore/

Please try that and see. If you still have problems, I can try to find time to help you.

**Comment by gordon-matt** [comment]
My apologies. I just came back to this project after a long time busy with other things and when I looked at this issue again, I realized there is a missing plugin for tinymce. It's actually there in an [old version of the code](https://github.com/gordon-matt/peachpie-responsive-file-manager/tree/bb2539b7f2630aa61fc7681e662979e765d58595/WebApplication/wwwroot/js/tinymce/plugins/responsivefilemanager).. It must have got lost somehow when I moved tinymce into libman instead of storing it directly in the repo. I will solve this in a future commit. For now, get it from the link I mentioned before and add it to your tinymce plugins directory.
